storage/rocksdb/myrocks_hotbackup.py

```diff
@@ -26,8 +26,8 @@
 rocksdb_files = ['MANIFEST', 'CURRENT', 'OPTIONS']
 rocksdb_data_suffix = '.sst'
 rocksdb_wal_suffix = '.log'
-exclude_files = ['master.info', 'relay-log.info', 'worker-relay-log.info',
-                 'auto.cnf', 'gaplock.log', 'ibdata', 'ib_logfile', '.trash']
+exclude_files = [b'master.info', b'relay-log.info', b'worker-relay-log.info',
+                 b'auto.cnf', b'gaplock.log', b'ibdata', b'ib_logfile', b'.trash']
```

Comment:
Can you describe the significance of this? It looks like a backwards/forwards utf8 coding/decoding.

Reply:
In python3, I don't know why,  db and filename is bytes , not str. 

Otherwise, an error will be reported.
After I modified it like this, it can run.

Reply:
see this https://github.com/oppia/oppia/wiki/Bytes-and-string-handling-in-Python-3

Reply:
I originally wanted to use mariadb-backup to back up myrocks, but found that it was impossible to back up myrocks without InnoDB.

For details, see:
https://lists.mariadb.org/hyperkitty/list/discuss@lists.mariadb.org/thread/7Z64ZTVLUESRSTU5QIED4OLNOCMT5DON/

I tried to use myrocks_hotbackup to backup and restore, but got stuck again. A .frm file exists for table i18n.uidHost, but that table is not registered in RocksDB

this is frustrating